recipe/meta.yaml

```diff
@@ -56,10 +55,10 @@ test:
     - pip
     - pytest
     - pytest-mock
-
+    - pyproject-metadata
```

Comment:
we alreay have this in run section, is it need also here?

Reply:
Yes, it is needed in both the run and host sections. Without it, the build automatically fails as it is heavily relied on within the host section for metadata generation.
` ModuleNotFoundError: No module named 'pyproject_metadata'
  error: subprocess-exited-with-error
  `